tests/baselines/reference/ES5For-ofTypeCheck10.errors.txt

```diff
@@ -20,4 +20,4 @@ tests/cases/conformance/statements/for-ofStatements/ES5For-ofTypeCheck10.ts(14,1
     
     for (var v of new StringIterator) { }
                   ~~~~~~~~~~~~~~~~~~
-!!! error TS2569: Type 'StringIterator' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.
\ No newline at end of file
+!!! error TS2495: Type 'StringIterator' is not an array type or a string type.
```

Comment:
> !!! error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the `lib` compiler option to es2015 or later.

The `Symbol.iterator` member no longer exists, because `Symbol.iterator` doesn't exist (this is an ES5 target project).